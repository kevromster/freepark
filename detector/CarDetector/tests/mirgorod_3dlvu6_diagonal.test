config = camera_nsk_mirgorod_3dlvu.txt
input = input/mirgorod_3dlvu6_diagonal.png
expected = expected/mirgorod_3dlvu6_diagonal.out
