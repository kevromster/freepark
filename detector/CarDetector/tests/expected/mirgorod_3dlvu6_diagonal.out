[
parking id: 24
spots count: 3

{ lat: 54.853807; lon: 83.040066; complexity: NEWBIE }
{ lat: 54.85379; lon: 83.040079; complexity: NEWBIE }
{ lat: 54.853748; lon: 83.04011; complexity: NEWBIE }
]

[
parking id: 25
spots count: 1

{ lat: 54.853669; lon: 83.040172; complexity: NEWBIE }
]

[
parking id: 26
spots count: 3

{ lat: 54.853801; lon: 83.039991; complexity: NEWBIE }
{ lat: 54.853758; lon: 83.04003; complexity: NEWBIE }
{ lat: 54.853716; lon: 83.040069; complexity: NEWBIE }
]

